src/components/email-flow/EmailFlowForm.tsx

```diff
@@ -0,0 +1,44 @@
+import { AccentText, BodyText, HeaderText, TinyText } from 'components/Text'
+import EmailProofForm from 'components/proofs/EmailProofForm'
+import classnames, { space, width } from 'classnames/tailwind'
+
+const proofLineContainer = classnames(space('space-y-4'), width('w-full'))
+
+export default function EmailFlowForm({
+  domain,
+  onUpdateDomain,
+}: {
+  domain?: string
+  onUpdateDomain: (data: { domain?: string }) => void
```

Comment:
We have a lot of `?` being passed around for something that should be a `string`